pytorch_lightning/tuner/lr_finder.py

```diff
@@ -40,8 +40,28 @@
     from tqdm import tqdm
 
 
+def __choose_lr_assigner(trainer, model: LightningModule) -> Callable[[Any], None]:
```

Comment:
Instead of returning the `Callable`, can you return the attribute string and then only call `lightning_setattr` in `_run_lr_finder_internally`?

Reply:
fixed